curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/create-strings-using-template-literals.english.md

```diff
@@ -36,7 +36,7 @@ This new way of creating strings gives you more flexibility to create robust str
 
 ## Instructions
 <section id='instructions'>
-Use template literal syntax with backticks to display each entry of the <code>result</code> object's <code>failure</code> array. Each entry should be wrapped inside an <code>li</code> element with the class attribute <code>text-warning</code>, and listed within the <code>resultDisplayArray</code>.
+Use template literal syntax with backticks to create an array of list element (<code>li</code>) strings. Each list element's text should be one of the <code>result</code> object's <code>failure</code> array elements and have a <code>class</code> attribute with the value <code>text-warning</code>. The <code>makeList</code> function should return the array of list item strings.
```

Comment:
Maybe it's just me, but the wording seemed a bit confusing -

> Each list element's text should be one of the <code>result</code> object's <code>failure</code> array elements

How about 

> Each list element's text should be one of the array elements of the <code>failure</code> property of the <code>result</code> object.

Reply:
@lasjorg What do you think about @Manish-Giri's latest suggestion?

Reply:
@RandellDawson Seem fine either way. The sentence quickly becomes convoluted and hard to follow no matter what. The "of the, of the, of the" seem a little hard to follow as well.

How about "from" the array "on" the object?

> Each list element's text should be one of the array elements from the `failure` property on the `result` object.